src/Subscriptions/resources/components/SubscriptionsRowActions.tsx

```diff
@@ -18,27 +18,56 @@ export function SubscriptionsRowActions({item, setUpdateErrors, parameters}) {
         return response;
     };
 
-    const deleteItem = async (selected) => await fetchAndUpdateErrors(parameters, '/delete', item.id, 'DELETE');
+    const deleteItem = async () => await fetchAndUpdateErrors(parameters, '/delete', item.id, 'DELETE');
+    const trashItem = async () => await fetchAndUpdateErrors(parameters, '/trash', item.id, 'DELETE');
+    const restoreItem = async () => await fetchAndUpdateErrors(parameters, '/untrash', item.id, 'POST');
 
-    const confirmDelete = (selected) => <p>{sprintf(__('Really delete donation #%d?', 'give'), item.id)}</p>;
+    const confirmDelete = () => <p>{sprintf(__('Really delete donation #%d?', 'give'), item.id)}</p>;
+    const confirmTrash = () => <p>{sprintf(__('Trash the following subscription #%d?', 'give'), item.id)}</p>;
+    const confirmRestore = () => <p>{sprintf(__('Restore the following subscription #%d?', 'give'), item.id)}</p>;
 
-    const confirmModal = (event) => {
+    const confirmDeleteModal = () => {
         showConfirmModal(__('Delete', 'give'), confirmDelete, deleteItem, 'danger');
     };
 
+    const confirmTrashModal = () => {
+        showConfirmModal(__('Trash', 'give'), confirmTrash, trashItem, 'warning');
+    };
+
+    const confirmRestoreModal = () => {
+        showConfirmModal(__('Restore', 'give'), confirmRestore, restoreItem, 'normal');
+    };
+
     return (
         <>
             <RowAction
                 href={`edit.php?post_type=give_forms&page=give-subscriptions&id=${item.id}`}
                 displayText={__('Edit', 'give')}
             />
```

Comment:
@alaca Let's keep the Edit option only for items not trashed. The default WP list table does not allow editing items in the trash status. 

<img width="1868" height="806" alt="image" src="https://github.com/user-attachments/assets/0c437089-8ce8-41e0-82ee-4b1023f77ea0" />